src/components/Tooltip/Tooltip.scss

```diff
@@ -12,11 +12,11 @@
         position: absolute;
     }
 
-    [x-placement^="top"] & {
+    [data-popper-placement^="top"] & {
         margin-bottom: var(--spacing-5x);
```

Comment:
shouldn't all the margins be removed? https://popper.js.org/docs/v2/migration-guide/#4-remove-all-css-margins

Reply:
@eszthoff I removed most of the margins, but I left positioning ones like this one. Popper.js proposal is to replace them with `offset` props which is not supported by IE and Edge. Since those margins are working fine and don't make any sort of conflicts and overwriting maybe it makes sense to leave them as it is?